tools/importer-rest-api-specs/components/dataapigeneratoryaml/revision_id.go

```diff
@@ -0,0 +1,25 @@
+package dataapigeneratoryaml
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v3"
```

Comment:
We might want this to be? (It's the same, but via the typical route, rather than gopkg)
```suggestion
	"github.com/go-yaml/yaml"
```
ref: https://pkg.go.dev/gopkg.in/yaml.v3#pkg-overview